cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/InitCommand.java

```diff
@@ -144,6 +144,16 @@ public void execute() {
             return;
         }
 
+        if (!ProjectUtils.validateInitialNumericsOfName(packageName)) {
```

Comment:
Init command only produces an empty Ballerina.toml. So I think we don't need to do any validations here. Instead, we should replace the disallowed characters when we derive the package name in ManifestBuilder

Reply:
What we should replace with numeric characters? 
Here if the user is giving initial `_`s we give it as an error. 
I followed the same approach for initial numeric characters.